doc/lightning-fundchannel.7.txt

```diff
@@ -0,0 +1,54 @@
+LIGHTNING-FUNDCHANNEL(7)
+========================
+:doctype: manpage
+
+NAME
+----
+lightning-fundchannel - Command for establishing a lightning channel.
+
+SYNOPSIS
+--------
+*fundchannel* 'id' 'satoshi'
+
+DESCRIPTION
+-----------
+The *fundchannel* RPC command opens a payment channel with a peer by commiting
+a funding transaction to the blockchain as defined in BOLT #2.  Once the
+transaction is confirmed, normal channel operations may begin.  Readiness is
```

Comment:
Suggest adding newlines at end of sentence.  That way, even if an earlier sentence is later amended, only that sentence needs to be reformatted and later sentences in the same paragraph need not be affected.  E.g.

    The *fundchannel* RPC command opens a payment channel with a peer by commiting
    a funding transaction to the blockchain as defined in BOLT #2.
    Once the transaction is confirmed, normal channel operations may begin.
    Readiness is indicated by *listfunds* reporting a `short_channel_id` for
    the channel.

Then later if we amend it:

    The *fundchannel* RPC is a totally radical command that opens a really cool payment
    channel with a peer by commiting a funding transaction to the blockchain as defined
    in the outrageously awesome BOLT #2.
    Once the transaction is confirmed, normal channel operations may begin.
    Readiness is indicated by *listfunds* reporting a `short_channel_id` for
    the channel.